### PR TITLE
update style and content home page

### PR DESCRIPTION
**Pull request by thomasmaas**
Screenshot 2022-12-20 at 13 55 48

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -3,76 +3,47 @@
...
 	</a>
-	<p class="lead mt-5">Kubernetes Native stuff that you want to use!</p>
-	{{< blocks/link-down color="primary" >}}
+	<p class="lead mt-10">Kuadrant bundles <a href="https://github.com/Kuadrant/limitador">Limitador</a>, the rate limiting service, and <a href="https://github.com/Kuadrant/authorino">Authorino</a>, our lightweight authorization server. The Kuadrant Operator, available from <a href="https://operatorhub.io/operator/kuadrant-operator">Operator Hub</a>, will take care of all the wiring and deployments of the services required for applying Rate Limits and Authorization Policies to your infrastructure.</p><p>Once installed, Policy Custom Resource Definitions let cluster operators apply a cluster-wide Rate Limit policy to protect the infrastructure from DDoS attacks; while application developers can use advanced features to control usages of their API endpoints based on API keys handed out to their users. Kuadrant supports a wide range of use cases, from simple Rate Limiting to ones that span across authorization and identity-based Rate Limits. Kuadrant leverages the <a href="https://gateway-api.sigs.k8s.io/">Kubernetes Gateway API</a></p>
```
Comment (guicassolato) [inline content/en/_index.html:14]: > application developers can use advanced features to control usages of their API endpoints based on API keys handed out to their users.

I think this is way too specific for the description. Readers may inadvertently think Kuadrant is only API key authentication.

Comment (guicassolato) [reply content/en/_index.html:14]:
```suggestion
	<p class="lead mt-10">Kuadrant bundles <a href="https://github.com/Kuadrant/limitador">Limitador</a>, the rate limiting service, and <a href="https://github.com/Kuadrant/authorino">Authorino</a>, our lightweight authorization agent. The Kuadrant Operator, available from <a href="https://operatorhub.io/operator/kuadrant-operator">Operator Hub</a>, will take care of all the wiring and deployments of the services required for applying Rate Limits and Authorization Policies to your infrastructure.</p><p>Once installed, Policy Custom Resource Definitions let cluster operators apply a cluster-wide Rate Limit policy to protect the infrastructure from DDoS attacks; while application developers can use advanced features to control usages of their API endpoints with tailor-made authentication and authorization rules and protocols. Kuadrant supports a wide range of use cases, from simple Rate Limiting to ones that span across authorization and identity-based Rate Limits. Kuadrant leverages the <a href="https://gateway-api.sigs.k8s.io/">Kubernetes Gateway API</a></p>
```

**Review by guicassolato — APPROVED** [review]

**Review by alexsnaps — APPROVED** [review]
Any particular reason you removed the link to the github discussion? Not that I disagree but wondered... 
Also, we can use that new background image as is, right? no (c) or need to be mentioning the author?

**Comment by thomasmaas** [comment]
- Found it confusing to have 2 links to GitHub plus discussions is too empty (at this moment) to be a nice landing page
- image is by unknown artist from 1500…

**Pull request merged**